packages/pds/src/config.ts

```diff
@@ -50,6 +50,8 @@ export interface ServerConfigValues {
   hiveApiKey?: string
   labelerDid: string
   labelerKeywords: Record<string, string>
+  unacceptableHandleWordsB64?: string
+  falsePositiveHandleWordsB64?: string
```

Comment:
nitpick: Loading this file would be better for transparency. Keep the env var though, the file should be a fallback used in prod!